### ISSUE TITLE
MathJax error on \left<a,b\right>

### ISSUE THREAD

**Issue by kno10** [plastex/plastex#280]
This is incorrectly turned into `\left{\lt}a,b\right{\rt}` by
https://github.com/plastex/plastex/blob/8dffc0281fe7b0a0ba6cc5498f597d0492f483cb/plasTeX/Base/LaTeX/Math.py#L58

In this case, `<` and `>` should be left alone / reproduced in the output as `&lt;` resp `&gt;`, or converted to `\langle` and `\rangle` for all I can tell. Changing this line to
```python
    return s.replace('<', r'&lt;').replace('>', r'&gt;')
```
works for me.

**Comment by PatrickMassot** [comment]
Thanks for this report. Could you please tell whether b3bcf6dfedb4060 fixes the issue for you?

**Comment by kno10** [comment]
How about `\Big<a,b\Big>`? There are more cases I guess, I don't think you want to enumerate them.

**Comment by kno10** [comment]
Can we maybe settle for the simple and working solution of 
```python
return s.replace('<', r'&lt;').replace('>', r'&gt;')
```

**Comment by mennucc** [comment]
for the record, to represent a scalar product in LaTeX you should use `\langle a,b\rangle` and not `<a,b>` because otherwise the spacing will be all wrong

**Comment by mennucc** [comment]
I checked... it seems that in LaTeX `\left` and `\right` and `\Big` automatically convert `<` to `\langle` and `>` to `\rangle` . So when using size modifiers,  `\langle a,b\rangle` and  `<a,b>` are rendered the same. But when not using them  `\langle a,b\rangle` and  `<a,b>` are quite different

**Comment by mennucc** [comment]
here is the test , if you are interested:
[rangle.zip](https://github.com/plastex/plastex/files/9990468/rangle.zip)

**Comment by kno10** [comment]
Well, the question is not on what is the right latex syntax for a particular meaning (such as scalar product), we may need to be able to parse/translate what we get by the user.

`\left<` and `\Big<` are valid and common latex math / MathJax, and the replacement produced by above substitution command does not work. Hence the solution to only do HTML escaping.

**Comment by PatrickMassot** [comment]
Closed in ffa1ec30269bcd98ce8fd2ad859ba8a832e33c46
